code/modules/mining/equipment_locker.dm

```diff
@@ -1134,12 +1138,15 @@
 		new /datum/data/mining_equipment("Sulphuric Acid",			/obj/item/weapon/reagent_containers/glass/beaker/sulphuric,        		500),
 		new /datum/data/mining_equipment("Brute First-Aid Kit",		/obj/item/weapon/storage/firstaid/brute,						   		600),
 		new /datum/data/mining_equipment("Advanced Scanner",		/obj/item/device/t_scanner/adv_mining_scanner,                     		800),
+		new /datum/data/mining_equipment("Kinetic Accelerator", 	/obj/item/weapon/gun/energy/kinetic_accelerator,               	   		750),
```

Comment:
Costs 1/3 as much as the super KA but doubles your damage.

Also even less reason for science to make you anything.